backend/core/illustration_service.py

```
@@ -0,0 +1,179 @@
+import json
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)